vocab/credentials/v2/vocabulary.yml

```diff
@@ -79,6 +85,12 @@ property:
     range: IRI
     comment: An entity about which claims are made.
 
+  - id: confidenceMethod
+    label: Confidence method
+    defined_by: https://w3c-ccg.github.io/confidence-method-spec/
```

Comment:
We reserved this in a table... do we expect that term definition to show up here? 

https://www.w3.org/2018/credentials#confidenceMethod

https://w3c.github.io/vc-data-model/#reserved-extension-points

Reply:
The current vocabulary includes entries for the other reserved terms already, and the definition in the spec explicitly refers to a URL in the credentials vocabulary. Ie., consistency requires to put this entry into the table, too.

Reply:
I think you are saying, we don't expect to see these URLs in the vocubulary. 

And defined by URL can change in the future if the working group decides to define reserved terms?

Reply:
I am saying there is no definition as of today. We are not yet at Rec and the reserved terms are somehow at risk. If they become "regular" than a proper URL must come to the fore.